src/features/auth/login.tsx

```diff
@@ -0,0 +1,19 @@
+import { useAuth0 } from '@auth0/auth0-react';
+import { Button } from '@mui/material';
+
+
+interface ILoginButtonProps {
```

Comment:
empty redundant block, don't use I for interfaces - old bad practice. And we don't need interfaces here, type for 99% cases